README.md

```diff
@@ -33,18 +36,62 @@ won't show up until after the next update.
             'api_key': os.getenv('TNS_API_KEY', ''),  # This is the API key for the associated BOT         
             'tns_base_url': 'https://sandbox.wis-tns.org/api',  # This is the sandbox URL. Use https://www.wis-tns.org/api for live submission.
             'group_names': ['bot_group', 'PI_group'],  # Optional List. Include if you wish to use any affiliated Group Names when reporting.
+            'filter_mapping': {
+                'o': 'Other',
+                'c': 'Clear',
+                ...
+            },  # Optional mapping from your reduced datum filter values to TNS filter options.
+            'default_authors': 'Foo Bar <foo@bar.com>'  # Optional default authors string to populate the author fields for tns submission. If not specified, defaults to saying "<logged in user> using <tom name>".
         },
     }
     ```
 
-5. Include the tom_tns URLconf in your project `urls.py`:
+    ### Submit to TNS through Hermes
+    If you want to submit to the TNS through [HERMES](https://hermes.lco.global), then you must first configure your [hermes account profile](https://hermes.lco.global/profile) to have your TNS credentials associated with your account. Then you must add your hermes credentials you `settings.py` in the `DATA_SHARING` section. This method of TNS submission will take precedence if `ENABLE_TNS` is set to True in your hermes `DATA_SHARING` details.
+
+    NOTE: If you don't set TNS credentials in your hermes account profile, TNS submission will still work but use the default Hermes_Bot account for TNS submission.
+
+
+   ```python
+    DATA_SHARING = {
+        ...
+        'hermes': {
+            'DISPLAY_NAME': 'hermes',
+            'BASE_URL': 'https://hermes-dev.lco.global/',
+            'HERMES_API_KEY': 'YourHermesAPITokenHere',
+            'DEFAULT_AUTHORS': 'String of default author list for your submissions heres',  # Optional default authors string to populate the author fields for tns submission. If not specified, defaults to saying "<logged in user> using <tom name>".
+            'USER_TOPICS': ['hermes.discovery', 'hermes.classification', ...]  # This is a list of hermes topics you will be allowed to share on. hermes.discovery and hermes.classification are automatically used for TNS submissions of those types.
+            'TNS_GROUP_NAMES': ['bot_group', 'PI_group'],  # Optional List. Include if you wish to use any affiliated Group Names when reporting to TNS.
+            'FILTER_MAPPING': {
+                'o': 'Other',
+                'c': 'Clear',
```

Comment:
It isn't really important, but the o and c bands are from the atlas survey and stand for orange and cyan respectively.

Reply:
The purpose of this readme is just to show an example setting... users should tailor it to their own data sources and mapping. But if there is a better example mapping you want me to put here please let me know (Cyan/Orange aren't tns filter names I think).